apps/bare-expo/package.json

```diff
@@ -37,8 +37,10 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.20.0",
+    "@dotlottie/react-player": "^1.6.19",
```

Comment:
Do we need this to be a dependency of bare-expo? It seems to be used only by NCL and dropping changes to package.json here didn't seem to impact bare-expo working.

Reply:
@aleqsio, you are right, they are not needed there 😅 
I've removed them and it seems to work. Good catch, thanks!
I've updated the PR!